frontends/web/src/api/lightning.ts

```diff
@@ -282,17 +282,17 @@ export type MetadataItem = {
 };
 
 export type NodeState = {
-  id: string;
-  blockHeight: number;
+  // id: string;
+  // blockHeight: number;
   channelsBalanceMsat: number;
-  onchainBalanceMsat: number;
-  utxos: UnspentTransactionOutput[];
-  maxPayableMsat: number;
-  maxReceivableMsat: number;
-  maxSinglePaymentAmountMsat: number;
-  maxChanReserveMsats: number;
-  connectedPeers: string[];
-  inboundLiquidityMsats: number;
+  // onchainBalanceMsat: number;
+  // utxos: UnspentTransactionOutput[];
+  // maxPayableMsat: number;
+  // maxReceivableMsat: number;
+  // maxSinglePaymentAmountMsat: number;
+  // maxChanReserveMsats: number;
+  // connectedPeers: string[];
+  // inboundLiquidityMsats: number;
```

Comment:
All this fields are not currently used in the FE. I guess it will be useful to get all the node info in the future, but atm this is dead code. We should review the whole FE API and delete the unnecessary code.

Reply:
IMO the type should always contain all data that is returned by the backend, even if we don't use it.

There seems to be useful information in here that we should consider showing somehow. i.e. maxPayableMsat or maxReceivableMsat, inboundLiquidityMsats.

Could you keep as is for now and remove/refactor later?